HW1.py

```diff
@@ -0,0 +1,104 @@
+wrong_alphabet = 'BbDdEeFfHhIiJjKkLlMmNnOoPpQqRrSsVvWwYyZz'
+while True:
+    print("Enter command")
+    command = str(input())
```

Comment:
`input` уже возвращает строку, преобразование при помощи `str` излишне.
```suggestion
    command = input()
```